dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -95,12 +95,11 @@ public static Path generate()
     // LinkedListMultimap preserves the key order
     ImmutableList<Path> classpath = ImmutableList.copyOf(jarToDependencyPaths.keySet());
 
-    // TODO(suztomo): choose entry point classes for reachability
-    ImmutableSet<Path> entryPoints = ImmutableSet.of(classpath.get(0));
+    // When checking a BOM, entry point classes are the ones in the artifacts listed in the BOM
+    List<Path> artifactJarsInBom = classpath.subList(0, managedDependencies.size());
+    ImmutableSet<Path> entryPoints = ImmutableSet.copyOf(artifactJarsInBom);
 
-    boolean onlyReachable = false;
```

Comment:
There should still be an option here, and the default should be false; that is, not limit ourselves to "reachable" classes.

Reply:
This change is to mark the unreachable classes. Nothing is to be filtered based on them.